examples/tags-in-searchbox/app.tsx

```diff
@@ -0,0 +1,366 @@
+/** @jsx h */
+import {
+  autocomplete,
+  AutocompleteComponents,
+  getAlgoliaResults,
+  getAlgoliaFacets,
+} from '@algolia/autocomplete-js';
+import {
+  AutocompleteInsightsApi,
+  createAlgoliaInsightsPlugin,
+} from '@algolia/autocomplete-plugin-algolia-insights';
+import { createTagsPlugin, Tag } from '@algolia/autocomplete-plugin-tags';
+import algoliasearch from 'algoliasearch';
+import { h, Fragment, render } from 'preact';
+import groupBy from 'ramda/src/groupBy';
+import insightsClient from 'search-insights';
+
+import '@algolia/autocomplete-theme-classic';
+
+import { ProductHit, TagExtraData } from './types';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+
+// @ts-expect-error type error in search-insights
+insightsClient('init', { appId, apiKey });
+
+const algoliaInsightsPlugin = createAlgoliaInsightsPlugin({ insightsClient });
+
+const tagsPlugin = createTagsPlugin<TagExtraData>({
+  getTagsSubscribers() {
+    return [
+      {
+        sourceId: 'brands',
+        getTag({ item }) {
+          return item;
+        },
+      },
+      {
+        sourceId: 'categories',
+        getTag({ item }) {
+          return item;
+        },
+      },
+    ];
+  },
+  transformSource() {
+    return undefined;
+  },
+  onChange({ tags }) {
+    requestAnimationFrame(() => {
+      const container = document.querySelector('.aa-InputWrapperPrefix');
+      const oldTagsContainer = document.querySelector('.aa-Tags');
+
+      const tagsContainer = document.createElement('div');
+      tagsContainer.classList.add('aa-Tags');
+
+      render(
+        <div className="aa-TagsList">
+          {tags.map((tag) => (
+            <TagItem key={tag.label} {...tag} />
+          ))}
+        </div>,
+        tagsContainer
+      );
+
+      if (oldTagsContainer) {
+        container.removeChild(oldTagsContainer);
+      }
+
+      container.appendChild(tagsContainer);
```

Comment:
Can we use [`replaceChild`](https://developer.mozilla.org/en-US/docs/Web/API/Node/replaceChild)?

Reply:
Addressed in 12cd15b9.